content/resources/operator/_index.md

```diff
@@ -112,7 +113,19 @@ One of the goals when providing an external reference is to figure out the level
 
 #### Creating Valuable Data Descriptions
 
-It's important to create clear and concise descriptions for each property in a dictionary.  The descriptions should be understandable by someone who is not familiar with a particular domain.  When available, including the unit of measure in parenthesis at the end of the description would be helpful in cases where the unit of measure is not included elsewhere in the description. When a clear description is not readily available, it's recommended that an external vocabulary such as NCIt be used as they offer definitions for terms from a plethora of domains.
+It's important to create clear and concise descriptions for each property in a dictionary. The descriptions should be understandable by someone who is not familiar with a particular domain. When available, including the unit of measure in parenthesis at the end of the description would be helpful in cases where the unit of measure is not included elsewhere in the description. When a clear description is not readily available, it's recommended that an external vocabulary such as NCIt be used as they offer definitions for terms from a plethora of domains.
+
+#### Selecting the Property Type
+
+The Gen3 model allows for a selection of property types that can be used to specify or limit values submitted for each property.
+The available types follow the [jsonschema basic types](https://cswr.github.io/JsonSchema/spec/basic_types/):
+- string; [example](https://github.com/uc-cdis/openaccess_dictionary/blob/master/gdcdictionary/schemas/drug.yaml#L274)
+- boolean; [example](https://github.com/uc-cdis/openaccess_dictionary/blob/master/gdcdictionary/schemas/drug.yaml#L79)
+- number; [example](https://github.com/uc-cdis/openaccess_dictionary/blob/master/gdcdictionary/schemas/drug.yaml#L354); can be float, long, or double as shown [here](https://json-schema.org/understanding-json-schema/reference/numeric.html).
+- integer; [example](https://github.com/uc-cdis/openaccess_dictionary/blob/master/gdcdictionary/schemas/drug.yaml#L418)
+- enum; [example](https://github.com/uc-cdis/openaccess_dictionary/blob/master/gdcdictionary/schemas/drug.yaml#L401)
+- array; [example](https://github.com/uc-cdis/openaccess_dictionary/blob/master/gdcdictionary/schemas/expression_result.yaml#L150)
+- regex patterns (including strings, integers, and numbers to modify the format); [example](https://github.com/uc-cdis/openaccess_dictionary/blob/master/gdcdictionary/schemas/_definitions.yaml#L103)
```

Comment:
replace `master` with `415bf61707ba55a6772423c461da3f16eef9c451` (the latest commit) in all these links. If you just use "master", the links won't point to the same line anymore when the file contents change